### ISSUE TITLE
A problem occurred configuring project ':google_api_availability'.

### ISSUE THREAD

**Issue by antoninobajeli** [BaseflowIT/flutter-geolocator#130]
## 🐛 Bug Report

Including the plugin I get the following error during the compilation time.

* Error running Gradle:
Exit code 1 from:  myapplication/android/gradlew app:properties:

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':google_api_availability'.
> Failed to notify project evaluation listener.
   > java.lang.AbstractMethodError (no error message)

**Version:** 2.1.0

**Comment by natezhengbne** [comment]
The same problem occurred when I add geolocator: '^2.1.0' to the dependencies

**Comment by arleyduarte** [comment]
The same problem

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':google_api_availability'.
> Failed to notify project evaluation listener.
   > java.lang.AbstractMethodError (no error message)

**Comment by mvanbeusekom** [comment]
@antoninobajeli, @natezhengbne and @arleyduarte, can you send over the contents of your `build.gradle` and/or `gradle-wrapper.properties` files?

This is to make sure you are using the correct version of Gradle (and potentially Kotlin). More information can also be found in the posts below:

- Issue #127 
- [flutter/flutter#22614](https://github.com/flutter/flutter/issues/22614)

**Comment by natezhengbne** [comment]
@mvanbeusekom I think the problem is the version of Gradle. My project is work well.
Thank you.

**Comment by MrMagloire** [comment]
Please @mvanbeusekom, I make all change of my Gradle and kotlin but i have the same error

**Comment by glachac** [comment]
I see the same issue.  Do you use firebase?  I think there is a conflict with the firebase version and the google api versions.

**Comment by MrMagloire** [comment]
Yes I use cloud_firestore and map_view

**Comment by antoninobajeli** [comment]
@mvanbeusekom I thank you very much.
You helped me in finding the problem solution! 
In my case it has been not-sufficient changing the version numbers in build.gradle of gradle,kotlin and/ gradle-wrapper.properties. May be somthing is going wronk in my developer environment.

I solved creating a brand new project, than I changed the default build.gradle,kotlin and/or gradle-wrapper.properties version numbers.


Thanks very much !!!
